app/config/redis.py

```
@@ -0,0 +1,244 @@
+                socket_timeout=self.config.socket_timeout,
+                socket_connect_timeout=self.config.socket_connect_timeout,
+                max_connections=self.config.pool_size,
+                retry_on_timeout=True,
+                health_check_interval=30,
+            )
+
+            # Test connection
+            await self._single_client.ping()
+            logger.info("Single Redis connection established successfully")
+
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)